### ISSUE TITLE
Does not animate on 0.59.1

### ISSUE THREAD

**Issue by philipaarseth** [bartgryszko/react-native-circular-progress#181]
Hi,

I just upgraded to 0.59.1 and the circular-progress **stopped animating**. It renders the starting position just fine w/wo prefill.

The first of both instances is what I used to have, the second is from the example from the readme which doesn't work either.

Has anybody noticed this? and or knows how to fix it?

```
componentDidMount(){
    this.refs.circleRef.animate(this.state.fill, 4000, Easing.quad);
    this.circularProgress.animate(100, 8000, Easing.quad);
  }
```

```
<AnimatedCircularProgress
          ref="circleRef"
          size={200}
          width={10}
          rotation={0}
          lineCap="round"
          fill={this.state.fill}
          tintColor={colors.primary}
          backgroundColor={colors.inactive}
          style={{marginBottom: 20}}>
          {
            (fill) => (
              <Text style={styles.points}>
                {this.state.status}{this.state.analyte}
              </Text>
            )
          }
        </AnimatedCircularProgress>

        <AnimatedCircularProgress
        ref={(ref) => this.circularProgress = ref}
  size={120}
  width={15}
  fill={100}
  tintColor="#00e0ff"
  onAnimationComplete={() => console.log('onAnimationComplete')}
  backgroundColor="#3d5875" />
```

environment: no expo
```
    "react": "16.8.3",
    "react-native": "0.59.1",
    "react-native-circular-progress": "^1.1.0",
    "react-native-svg": "^9.3.3",
```

testing on iOS 12.1.3

**Comment by VasilUrda** [comment]
Same Issue I have!!

**Comment by lhh-sofy** [comment]
me to!!

  "react": "16.8.3",
    "react-native": "0.59.0",
    "react-native-circular-progress": "^1.1.0",
    "react-native-svg": "^9.3.3",

**Comment by lhh-sofy** [comment]
> Hi,
> 
> I just upgraded to 0.59.1 and the circular-progress **stopped animating**. It renders the starting position just fine w/wo prefill.
> 
> The first of both instances is what I used to have, the second is from the example from the readme which doesn't work either.
> 
> Has anybody noticed this? and or knows how to fix it?
> 
> ```
> componentDidMount(){
>     this.refs.circleRef.animate(this.state.fill, 4000, Easing.quad);
>     this.circularProgress.animate(100, 8000, Easing.quad);
>   }
> ```
> ```
> <AnimatedCircularProgress
>           ref="circleRef"
>           size={200}
>           width={10}
>           rotation={0}
>           lineCap="round"
>           fill={this.state.fill}
>           tintColor={colors.primary}
>           backgroundColor={colors.inactive}
>           style={{marginBottom: 20}}>
>           {
>             (fill) => (
>               <Text style={styles.points}>
>                 {this.state.status}{this.state.analyte}
>               </Text>
>             )
>           }
>         </AnimatedCircularProgress>
> 
>         <AnimatedCircularProgress
>         ref={(ref) => this.circularProgress = ref}
>   size={120}
>   width={15}
>   fill={100}
>   tintColor="#00e0ff"
>   onAnimationComplete={() => console.log('onAnimationComplete')}
>   backgroundColor="#3d5875" />
> ```
> environment: no expo
> 
> ```
>     "react": "16.8.3",
>     "react-native": "0.59.1",
>     "react-native-circular-progress": "^1.1.0",
>     "react-native-svg": "^9.3.3",
> ```
> testing on iOS 12.1.

> Hi,
> 
> I just upgraded to 0.59.1 and the circular-progress **stopped animating**. It renders the starting position just fine w/wo prefill.
> 
> The first of both instances is what I used to have, the second is from the example from the readme which doesn't work either.
> 
> Has anybody noticed this? and or knows how to fix it?
> 
> ```
> componentDidMount(){
>     this.refs.circleRef.animate(this.state.fill, 4000, Easing.quad);
>     this.circularProgress.animate(100, 8000, Easing.quad);
>   }
> ```
> ```
> <AnimatedCircularProgress
>           ref="circleRef"
>           size={200}
>           width={10}
>           rotation={0}
>           lineCap="round"
>           fill={this.state.fill}
>           tintColor={colors.primary}
>           backgroundColor={colors.inactive}
>           style={{marginBottom: 20}}>
>           {
>             (fill) => (
>               <Text style={styles.points}>
>                 {this.state.status}{this.state.analyte}
>               </Text>
>             )
>           }
>         </AnimatedCircularProgress>
> 
>         <AnimatedCircularProgress
>         ref={(ref) => this.circularProgress = ref}
>   size={120}
>   width={15}
>   fill={100}
>   tintColor="#00e0ff"
>   onAnimationComplete={() => console.log('onAnimationComplete')}
>   backgroundColor="#3d5875" />
> ```
> environment: no expo
> 
> ```
>     "react": "16.8.3",
>     "react-native": "0.59.1",
>     "react-native-circular-progress": "^1.1.0",
>     "react-native-svg": "^9.3.3",
> ```
> testing on iOS 12.1.3

Have you solved this problem?

**Comment by andriinegustorov** [comment]
+ 1 on iOS only

**Comment by lichstam** [comment]
upgrade to latest `react-native-svg`

**Comment by lichstam** [comment]
`9.3.5`

**Comment by m-ochyra** [comment]
> `9.3.5`
It works for me.

**Comment by Jacse** [comment]
This is due to https://github.com/react-native-community/react-native-svg/issues/973. As other people have figured out, it can be fixed by upgrading to `react-native-svg>=9.3.4`

**Comment by moahammadalt** [comment]
react-native-svg>=9.4.0 worked for me
